starter-project-web-vue2/pages/zerubabel/_id.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <v-container>
+        <v-row>
+          <v-btn
+          class="ma-2"
+          color="primary"
+          dark
+        >
+          <nuxt-link to="/zerubabel" class="link">
+            <v-icon
+              dark
+              left
+            >
+              mdi-arrow-left
+            </v-icon>Back
+          </nuxt-link>
+        </v-btn>
+        </v-row>
+        <v-row class="pa-6 d-flex flex-column">
+          <v-row class="d-flex flex-column">
+            <h1 class="title">{{blog.title}}</h1>
+            <h3 class="author">Author: {{blog.author}}</h3>
+          </v-row>
+          <v-row>
+            <p class="desc">{{blog.description}}</p>
+          </v-row>
+        </v-row>
+    </v-container>
+</template>
+
+<script>
+import { mapState} from 'vuex'
+
+export default {
+  data(){
+    return {
+      id: null,
+      blog: null
+    }
+  },
+  computed: {
+    ...mapState('zerubabel', ['blogs']),
+    },
+  created(){
+    const path = this.$route.fullPath.split('/')
+    this.id = path[path.length - 1]
+    this.blogs.forEach(element => {
+      if(element._id === parseInt(this.id)){
+        this.blog = element
+      }
+    });
+  }
+}
+</script>
+
+<style scoped>
+.link{
+  text-decoration: none;
```

Comment:
also here, using vuetify classes is preferred. for coloring check [here](https://vuetifyjs.com/en/styles/colors/)  for font styles,size check [here](https://vuetifyjs.com/en/styles/text-and-typography/)